authz/providers/azure/rbac/rbac.go

```diff
@@ -293,16 +293,16 @@ func (a *AccessInfo) CheckAccess(request *authzv1beta1.SubjectAccessReviewSpec)
 	glog.V(7).Infof("checkaccess response: %s, Configured ARM call limit: %d", string(data), a.armCallLimit)
 	if resp.StatusCode != http.StatusOK {
 		glog.Errorf("error in check access response. error code: %d, response: %s", resp.StatusCode, string(data))
-		// metrics for calls with StatusCode >= 300 
+		// metrics for calls with StatusCode >= 300
 		if resp.StatusCode >= http.StatusMultipleChoices {
-                        glog.V(10).Infoln("Closing idle TCP connections.")
-                        if resp.StatusCode == http.StatusTooManyRequests {
-                            checkAccessThrottled.Inc()
-                        }
+			glog.V(10).Infoln("Closing idle TCP connections.")
```

Comment:
this should also go inside 429 if loop along with closing connection.

Reply:
done